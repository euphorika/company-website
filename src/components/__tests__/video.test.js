import React from "react"
import { shallow } from "enzyme"
import toJson from 'enzyme-to-json'

import Video from "../video"

describe("Video", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Video  video="/videos/design-video.m4v" poster="/videos/design-poster.jpg" />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
