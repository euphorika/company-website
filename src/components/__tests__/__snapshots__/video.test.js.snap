// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video renders correctly 1`] = `
<section
  className="videoContainer"
>
  <div
    className="innerContainer"
  >
    <video
      autoPlay={true}
      loop={true}
      muted={true}
      playsInline={true}
      poster="/videos/design-poster.jpg"
    >
      <source
        src="/videos/design-video.m4v"
        type="video/mp4"
      />
    </video>
  </div>
</section>
`;
